File: 240528/짝수인 것만 출력하기/print-even-number.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
    Scanner sc=new Scanner(System.in);

    int n=sc.nextInt();
    int[] arr=new int [100];
    int[] new_arr=new int [100];
    int cnt=0;

    for(int i=0;i<n;i++)
    arr[i]=sc.nextInt();

    for(int i=0;i<n;i++){
        if(arr[i]%2==0){
            new_arr[cnt++]=arr[i];
        }
    }
    for(int i=0;i<cnt;i++)
    System.out.print(new_arr[i]+" ");
    }
}

File: 240308/짝수인 것만 출력하기/print-even-number.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    int arr[100];
    int new_arr[100];
    int cnt=0;
    int n;
    cin>>n;

    for(int i=0;i<n;i++){
        cin>>arr[i];
          if(arr[i]%2==0) 
          new_arr[cnt++] = arr[i];
    }

    for(int i=0;i<cnt;i++){

        cout<<new_arr[i]<<" ";
    }
    return 0;
  
}